salt/modules/zypperpkg.py

```diff
@@ -2106,28 +2179,42 @@ def unhold(name=None, pkgs=None, **kwargs):
 
     targets = []
     if pkgs:
-        for pkg in salt.utils.data.repack_dictlist(pkgs):
```

Comment:
Why are you removing this?

Reply:
This for loop removes the versions from the `pkgs` list, but in `zypper` the package could be locked with the specified version